site/en/Test.md

```diff
@@ -0,0 +1,41 @@
+* High recognition rate of layout and text
+* Cost-effective use
+* Quick adaptability/troubleshooting
+* Modularity
+* Output in standard formats
+* Connection to existing workflows
+* Well documented interfaces
+* Word coordinates
+* Trainability
+* Extensive GT corpora
+
+High quality of text recognition is most important, whereas the other characteristics are only named each by some of the pilot libraries and should be considered as desired but subordinate optional features.
+
+
+## Evaluation of the software tests
+In order to ensure the comparability of the individual tests in the pilot libraries, a questionnaire distributed to the pilot libraries at the beginning of the test. In this questionnaire, the general conditions of the test run, e.g. the technical equipment used and the tested OCR-D processors, as well as the documentation of the software, interfaces, functionality and usability of the software, its possibilities for being integrated into existing workflows and the required output formats are recorded. Furthermore, recognition quality, functionality and usability, open requirements and positive features of the OCR-D software as well as the results of the test were to be described.
+
+During the test phase, the various options for installing the OCR-D software with and without a docker container were tried out and the software was successfully installed on a wide range of differently powerful, partly virtual servers. For non-Intel computers (ARM, PowerPC64) this was more complicated and time consuming, as individual Python packages were not executable on these computers and had to be adapted manually. The complete installation of all available OCR-D processors (``ocrd_all``), which was developed during the test phase, was confirmed as the simplest and least complex variant and was therefore the most recommendable. No pilot library integrated the OCR-D software In a workflow software such as Kitodo, as this would have meant too much work for a simple test run. 
+
+The usage of the numerous OCR-D processors was described as a challenge. Calling the processors was not so much of a problem than understanding their respective areas of application and, in particular, the selecting and combining the processors to form meaningful workflows. For the first test, besides the technical documentation of the software, there was no overall documentation on its use available yet, which is also aimed at users inexperienced in OCR. The requirements and wishes of the testers for such a documentation were taken into account when preparing the manuals, which are now available [in the user area of the OCR-D website](https://ocr-d.de/de/use.html). 
+
+The OCR-D software runs very stable, no library reported any aborts. All of the required output formats are already offered, whereas the few changes, which are still on the interfaces, are already planned for the further development of the prototype.
```

Comment:
aborts -> crashes

offered -> provided

whereas the few changes, which are still -> whereas requested changes on OCR-D's interfaces are already ...